app/types/remix-routes.d.ts

```diff
@@ -0,0 +1,241 @@
+declare module "remix-routes" {
```

Comment:
types for `$path` from `remix-routes`
According to the [docs](https://github.com/yesmeck/remix-routes), it could be generated in `watch` mode but I didn't manage to make it run with correct types (nested types was skipped). Seems that in `watch` mode `@remix-run/v1-route-convention` doesn't provide correct routes that `remix-routes` expect to see.
During build process everything works just fine 🤷‍♂️

@johnthecat what to hear your thoughts